cli/azd/cmd/init.go

```diff
@@ -98,6 +99,13 @@ func (i *initFlags) Bind(local *pflag.FlagSet, global *internal.GlobalCommandOpt
 		false,
 		"Initializes a new application from your existing code.",
 	)
+	local.BoolVarP(
+		&i.up,
+		"up",
+		"",
+		false,
+		"Provision and deploy to Azure after initializing the project from a template.",
+	)
```

Comment:
We may need to also update the [`TestUsage-azd-init.snap`](https://github.com/Azure/azure-dev/blob/main/cli/azd/cmd/testdata/TestUsage-azd-init.snap) file - some instructions below:

https://github.com/Azure/azure-dev/blob/0340065907dee7ca71cd68ed132033cabb38cbce/cli/azd/test/snapshot/snapshot.go#L68-L70

Reply:
Updated